### ISSUE TITLE
Version support

### ISSUE THREAD

**Issue by Azaret** [uxsolutions/bootstrap-datepicker#2061]
As far as I know we have no guidelines on version support.
Currently we have a branch for `1.5`, `1.6` and soon `1.7`, last update on `1.5` is 2015-12-28 and `1.6` is 2016-08-05.
I think it's time to have some clear statement about support.

**Comment by Azaret** [comment]
My opinion.
The truth is that we are not much people and not much active for proper long term support of multiple versions. I'm very tempted of dropping supports on at least `1.5` and maybe `1.6` too.
We can keep long term support on `1.7` for bug fix, since we might introduce breaking changes on `2.0` it's more than fair to keep support for those who wont jump on the new version right away.

**Comment by acrobat** [comment]
I don't think we should pick a version as a LTS, the datepicker is released according to semver so each minor release (major.minor.patch) is backwards compatible. My idea is that each time latest minor is the supported version. 
So if someone encounters a bug in version 1.5 and it is fixed in 1.6 they should upgrade to that version. If 1.6 is not released we could fix this issue in 1.5.x. This way we can keep a fast release cycle which every user can follow as we don't introduce bc breaks in minor version.

For the switch to 2.0 I indeed think we should support 1.7 for some time, especially as we will introduce some bc breaks in 2.0. When 2.0 is released we have 2 supported version 1.7 and 2.0. After each bugfix the 2.x supported version moves (so 1.7 and 2.1, 1.7 and 2.2, ...)

This somewhat my idea of supported versions, but of course to be discussed!

**Comment by Azaret** [comment]
Semver is the theory, the elephant in the room is that we made BCs in every minor releases.
I'm still on `1.3.1` at work for this reason, I'm just not ready or don't trust the changes made.
But I guess we are on the same page, add to that we are not enough people to maintain LTS, but yet I still feel guilt asking for someone on `1.5` to update to `1.6` because his bug is solved in the latest, but I know that he will have other bugs because `1.7` exists and few bugs fixes have been made for `1.6` on it....
God forgive us 😅

**Comment by acrobat** [comment]
Yeah the idea about not breaking bc is good in 1.x but will I think we can strictly do it in 2.x as we now the code base more and have written more code our self.
